### PR TITLE
Using --enablerepo='lsst-ts*' to specifiy the repo

### PR DESCRIPTION
**Pull request by rbovill**
…onda package' stage in Jenkinsfile.conda. Added a doc statement to the waitUntil loop.

### CODE REVIEW

File: Jenkinsfile.conda [hunk 1]
```diff
@@ -7,6 +19,7 @@ pipeline {
     parameters {
         string(defaultValue: '5.1.0', description: 'The XML Version, exclude any preceeding "v" characters: X.Y.Z', name: 'xml_version')
         string(defaultValue: '4.1.2', description: 'The ts_sal version, exclude any preceeding "v" characters: X.Y.Z', name: 'sal_version')
+        choice choices: ['Bleed', 'Daily', 'Release'], description: 'The upstream build type (Bleed, Daily or Release). This determines from where to pull the RPM.', name: 'build_type'
```
Comment (tribeiro) [inline Jenkinsfile.conda:22]: Should default be "Release"?

File: Jenkinsfile.conda [hunk 2]
```diff
@@ -31,20 +45,31 @@ pipeline {
                 script {
                     sh """echo 'The XML version: ${params.xml_version}'"""
                     sh """echo 'The SAL version: ${params.sal_version}'"""
+                    sh """echo 'The BuildType: ${params.build_type}'"""
+                    if ( params.build_type == "Bleed" ) {
+                        flag = "lsst-ts-bleed"
```
Comment (tribeiro) [inline Jenkinsfile.conda:50]: Consider renaming this attribute to something more informative, e.g.; `ts_repo`

**Review by tribeiro — APPROVED** [review]
This is a very clean and elegant solution! I made two inline comments I hope you will consider before merging. One is to make "Release" the default option for `build_type` and the other is to rename `flag` to something more informative, e.g. `ts_repo`.

Also, remember to rebase and squash commits before merging.. :)

File: Jenkinsfile.conda (hunk 1 above)
Comment (rbovill) [reply Jenkinsfile.conda:22]: Switched the order to ['Release', 'Daily', 'Bleed'], which makes Release (the first option) the default.

File: Jenkinsfile.conda (hunk 2 above)
Comment (rbovill) [reply Jenkinsfile.conda:50]: Changed it to rpm_repo.

**Pull request merged**